solr/core/src/java/org/apache/solr/api/V2HttpCall.java

```diff
@@ -362,14 +363,15 @@ private void invokeJerseyRequest(CoreContainer cores, SolrCore core, Application
 
       // Set properties that may be used by Jersey filters downstream
       containerRequest.setProperty(SOLR_QUERY_REQUEST_KEY, solrReq);
-      containerRequest.setProperty(SolrRequestAuthorizer.CORE_CONTAINER_PROP_NAME, cores);
-      containerRequest.setProperty(SolrRequestAuthorizer.HTTP_SERVLET_REQ_PROP_NAME, req);
-      containerRequest.setProperty(SolrRequestAuthorizer.REQUEST_TYPE_PROP_NAME, requestType);
-      containerRequest.setProperty(SolrRequestAuthorizer.SOLR_PARAMS_PROP_NAME, queryParams);
-      containerRequest.setProperty(SolrRequestAuthorizer.COLLECTION_LIST_PROP_NAME, collectionsList);
-      containerRequest.setProperty(SolrRequestAuthorizer.HTTP_SERVLET_RSP_PROP_NAME, response);
+      containerRequest.setProperty(SOLR_QUERY_RESPONSE_KEY, rsp);
```

Comment:
Should we static import all the keys, or use the `RequestContextConstants.SOLR_QUERY_RESPONSE_KEY`?

Reply:
Personally I prefer the current, static-import syntax.  IMO fully qualifying every last constant adds "noise" without any extra "signal".  (Since it's trivial to lookup the origin of these constants in an IDE)

But if you or anyone else feels differently I'm happy to change it.